src/ameba/cli/schema.cr

```diff
@@ -0,0 +1,99 @@
+require "./cmd"
+
+schema = JSON.build(2) do |builder|
+  builder.object do
+    builder.field("$schema", "http://json-schema.org/draft-07/schema#")
+    builder.field("title", ".ameba.yml")
+    builder.field("description", "Configuration rules for the Crystal language Ameba linter")
+    builder.field("type", "object")
+    builder.field("additionalProperties", false)
+
+    builder.string("definitions")
+    builder.object do
+      builder.string("Severity")
+      builder.object do
+        builder.field("type", "string")
+        builder.string("enum")
+        builder.array do
+          Ameba::Severity.values.each do |value|
+            builder.string(value.to_s)
+          end
+        end
+      end
+
+      builder.string("BaseRule")
+      builder.object do
+        builder.field("type", "object")
+
+        builder.string("properties")
+        builder.object do
+          builder.string("SinceVersion")
+          builder.object do
+            builder.field("type", "string")
+          end
+
+          builder.string("Enabled")
+          builder.object do
+            builder.field("type", "boolean")
+            builder.field("default", true)
+          end
+
+          builder.string("Severity")
+          builder.object do
+            builder.field("$ref", "#/definitions/Severity")
+            builder.field("default", Ameba::Severity.default_severity.to_s)
```

Comment:
I'm sorry, that should have been this:

```suggestion
            builder.field("default", Ameba::Rule::Base.default_severity.to_s)
```

That's also general, since the default changes by the group... but it's better than nothing, I guess...